lib/bin/run_test.js

```diff
@@ -63,6 +63,23 @@ var _ = require('lodash');
 var co = require('co');
 var exit = require('exit');  // process.exit that works on Windows
 var promiseUtil = require('../promise_util');
+var cleaned = false;
+
+function cleanup(exitCode) {
+  if (cleaned) {
+    return;
+  }
+  cleaned = true;
+  require('ps-tree')(process.pid, function (err, children) {
```

Comment:
Any reason to not require this at the top? This is going to be used all the time pretty much, right?